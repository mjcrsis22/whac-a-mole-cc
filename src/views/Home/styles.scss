.home-container {
  .home-logo {
    display: block;
    height: 6rem;
    padding: 1.5rem;
  }
}
