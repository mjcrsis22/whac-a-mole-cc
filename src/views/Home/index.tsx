import AccessForm from '../../components/AccessForm';
import moleLogo from '../../assets/mole.png';
import './styles.scss';

function Home() {
  return (
    <div className="home-container">
      <img className="home-logo" src={moleLogo} alt="" />
      <AccessForm />
    </div>
  );
}

export default Home;
