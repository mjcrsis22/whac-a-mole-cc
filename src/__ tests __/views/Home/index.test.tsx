import { render } from '@testing-library/react';
import Home from '../../../views/Home';

jest.mock('../../../components/AccessForm', () => () => <>Cmp AccessForm</>);

describe('Home component', () => {
  afterEach(() => {
    jest.clearAllMocks();
  });
  test('Renders correctly', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
