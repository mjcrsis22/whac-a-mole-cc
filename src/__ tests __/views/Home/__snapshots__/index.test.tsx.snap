// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component Renders correctly 1`] = `
<div>
  <div
    class="home-container"
  >
    <img
      alt=""
      class="home-logo"
      src="test-file-stub"
    />
    Cmp AccessForm
  </div>
</div>
`;
